spec/controllers/admin_controller_spec.rb

```diff
@@ -13,23 +13,33 @@
   include Devise::Test::ControllerHelpers
 
   # this will bypass Pundit policy access checks so logging in is not necessary
-  before(:each) { Warden.test_mode! }
+  before(:each) {Warden.test_mode!}
```

Comment:
I don't think so for inline blocks. (I don't think we've been enforcing it.)  But that is actually the style I prefer so I'll change them.